script/build_plugin.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Can we add `set -ex` here to ensure that any returned errors stop the script? It will make for easier debugging if necessary.